packages/paste-core/components/code-block/src/CodeBlockTabList.tsx

```diff
@@ -1,9 +1,72 @@
-import { Box } from "@twilio-paste/box";
+import { Box, safelySpreadBoxProps } from "@twilio-paste/box";
 import type { BoxProps } from "@twilio-paste/box";
-import { TabList } from "@twilio-paste/tabs";
-import type { TabListProps } from "@twilio-paste/tabs";
+import { css, styled } from "@twilio-paste/styling-library";
+import { type TabListProps, TabsContext } from "@twilio-paste/tabs";
+import { TabPrimitiveList } from "@twilio-paste/tabs-primitive";
+import { type ThemeShape, useTheme } from "@twilio-paste/theme";
 import * as React from "react";
 
+const ShadowLeft = styled.div(({ bgColor, left }: { bgColor: string; left: string }) => {
+  return css({
+    content: "' '",
+    position: "absolute",
+    left,
+    top: 0,
+    bottom: 0,
+    width: "0px",
+    pointerEvents: "none",
+    boxShadow: `6px 12px 12px 2px ${bgColor}`,
```

Comment:
Used absolute values (not design tokens) as we do not currently have anything for negative 6 like in the below example. This would also need to be a new boxShadow variable. 

This is for the edge shadow on the scroll bar so seems like it's an edge case and not standard on any other components. Happy to update design tokens if we feel the need!